libs/vue/README.md

```diff
@@ -1,7 +1,136 @@
+</div>
 
-Run `ng test vue` to execute the unit tests via [Jest](https://jestjs.io).
+## Nx Dependency Graph Support
+
+**Nx's dep-graph does not support `.vue` files.** To patch support for `.vue` files, add the following npm script to your `package.json`:
+
+```
+"postinstall": "node node_modules/@nx-plus/vue/patch-nx-dep-graph.js"
+```
+
+**Help us!** We dislike this hack just as much as you do. Please give this Nx [issue](https://github.com/nrwl/nx/issues/2960) a 👍 so that we can remove this hack in the future.
+
+## Getting Started
+
+### Add the Plugin
+
+```
+# With npm
+npm install --save-dev @nx-plus/vue
+
+# With yarn
+yarn add --dev @nx-plus/vue
+```
+
+### Generate a Vue application
+
+```
+nx g @nx-plus/vue:app my-app
+```
+
+### Serve the application
+
+```
+nx serve my-app
+```
+
+### Build the application
+
+```
+nx build my-app
+```
+
+## Usage
+
+### App schematic
+
+`nx g @nx-plus/vue:app <name> [...options]`
+
+| Arguments | Description           |
+| --------- | --------------------- |
+| `<name>`  | The name of your app. |
+
+| Options            | Default   | Description                                    |
+| ------------------ | --------- | ---------------------------------------------- |
+| `--tags`           | -         | Tags to use for linting (comma-delimited).     |
+| `--directory`      | `apps`    | A directory where the project is placed.       |
+| `--style`          | `css`     | The file extension to be used for style files. |
+| `--unitTestRunner` | `jest`    | Test runner to use for unit tests.             |
+| `--e2eTestRunner`  | `cypress` | Test runner to use for end to end (e2e) tests. |
+| `--routing`        | `false`   | Generate routing configuration.                |
+| `--skipFormat`     | `false`   | Skip formatting files.                         |
+
+### Vuex schematic
+
+`nx g @nx-plus/vue:vuex <project> [...options]`
+
+| Arguments   | Description                                            |
+| ----------- | ------------------------------------------------------ |
+| `<project>` | The name of the project you would like to add Vuex to. |
+
+| Options        | Default | Description            |
+| -------------- | ------- | ---------------------- |
+| `--skipFormat` | `false` | Skip formatting files. |
+
+### Dev server builder
+
+`nx serve my-app [...options]`
+
+| Options           | Default   | Description                                                |
+| ----------------- | --------- | ---------------------------------------------------------- |
+| `--open`          | `false`   | Open browser on server start.                              |
+| `--copy`          | `false`   | Copy url to clipboard on server start.                     |
+| `--stdin`         | `false`   | Close when stdin ends.                                     |
+| `--optimization`  | `false`   | Enables optimization of the build output.                  |
```

Comment:
Should we add the default value in the schema? It's currently empty.

Reply:
That's intentional because `optimization` overrides the browser target default.

https://github.com/ZachJW34/nx-plus/blob/master/libs/vue/src/builders/dev-server/builder.ts#L60